mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -288,6 +286,11 @@ def runConfigWithMIOpenDriver(commandLine, envs):
         p1.kill()
         print("MIOpen benchmark timed out")
         outs, errs = p1.communicate()
+    else:
+        outs=outs.decode()
+        # Extract Elapsed time in ms
+        elapsedTimeInMs = outs.split("Elapsed: ")[1].split("ms (average)")[0]
```

Comment:
Several comments:
 - Please leave a more detailed comment on what format is expected here
 - Needs better error handling: Could you check if the elapsed time will be populated when MIOpenDriver fail to run? We do have some int8 nhwc conv unsupported by MIOpenDriver and we want to make sure that doesn't break this script.

Reply:
@jerryyin Can you elaborate about the unsupported configs?
MIOpenDriver.py auto generates different combinations of {directions, data_types, layouts} but it skips int8 non-forward convolutions, i.e. only convint8 -F 1 ... is used as configs.  Does this setup still contains configurations that are not supported by MIOpenDriver?
MIOpenDriver only generates one run for int8 nhwc:
`convint8 -F 1 -f NHWC -I NHWC -O NHWC -n 256 -c 256 -H 14 -W 14 -k 256 -y 3 -x 3 -p 1 -q 1 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1`

Reply:
@zhanglx13 We used two different ways to build MIOpen binary. When `MIOPEN_USE_MLIR=0` setup, all NHWC configs will fail. Please take a look at nightly results int8 + NHWC section to observe this more clearly. Because of that scenario, we want to make sure this script does not give up if the pattern is not observed.

Reply:
@jerryyin when MIOpenDriver fails to run, do you want to remove the whole table entry for this config or just the TFlops field of the table entry?

Reply:
or put NaN as the TFlops?

Reply:
In the very first report (miopen_tuned/untuned_perf.csv), empty string is used in the TFlops column if the config fails. The empty strings are further "processed" in later reports, such as the [performance report for mlir-nightly-all](http://rocmhead.amd.com:8080/job/MLIR/job/mlir-nightly-all/395/Performance_20report/).

Reply:
@zhanglx13 Just FYI I kicked off a perf report run on your 1d85eed commit [here](http://rocmhead.amd.com:8080/blue/organizations/jenkins/MLIR%2Fmlir/detail/PR-657/5/pipeline/360) and looks like it is bumping into some computation failures. You can do the same to regress your newer commit 60e2eb3

Reply:
1/2: Make sense, thanks for your explanation. I think with gfx908 it is less likely to hit such timeout. 

3: Please ping me when you are on site so I can demo it to you. Kicking off a perf run is generally not required. Though for this particular PR, it is going to be helpful to see at least one passing perf run.

Reply:
@jerryyin I just realized that the timeout happened on gfx908. I tried it again with `60e3eb3` and it still timed out on gfx908. Were you trying to say it is less likely to happen on gfx90a?

Reply:
@zhanglx13 Correct, things with gfx90a is that its filesystem is slow, but this isn't an issue on gfx908 in general. I just help you kicked off another run.

BTW, my observation is both failures happen in the same host of `ixt-hq-99`, it can be due to a host specific failure. This usually wind into one of the two roads:
 - Reboot the node (or ask Omkar to do it)
 - (Ask Omkar the login credential of the node), then see if you can reproduce it in that machine.

Reply:
Re the on-failure question: existing code puts in a NaN

Reply:
@zhanglx13 According to performance report [here](http://rocmhead.amd.com:8080/job/MLIR/job/mlir/job/PR-657/13/Performance_20report/), MLIR vs MIOpen
 - MLIR tflops for i8 should display values
 - Below, in MLIR perf test per kernel based values, the i8 columns for mlir should also have values

Reply:
@jerryyin I tried the given config and I got the error message: Error: only single f32 function result supported
And this is how I did the experiment:
1. run `./bin/MIOpenDriver.py convint8 -F 1 -f NHWC -I NHWC -O NHWC -n 256 -c 256 -H 14 -W 14 -k 256 -y 3 -x 3 -p 1 -q 1 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1` and I printed out the command line options for miopen-gen, which is `--operation conv2d -t i8 -x2 --fil_layout kyxc --in_layout nhwc --out_layout nhwk --batchsize 256 --in_channels 256 --in_h 14 --in_w 14 --out_channels 256 --fil_h 3 --fil_w 3 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 1 --padding_w 1`
2. `Then` I run miopen-gen | mlir-miopen-driver | mlir-rocm-runner as follows
`./bin/miopen-gen -ph ${miopen_gen_CMD} | ./bin/mlir-miopen-driver -c | ./bin/mlir-rocm-runner --shared-libs=./external/llvm-project/llvm/lib/libmlir_rocm_runtime.so,./lib/libconv-validation-wrappers.so,./external/llvm-project/llvm/lib/libmlir_runner_utils.so', '--entry-point-result=void` where ${miopen_gen_CMD} is the above miopen-gen command option
3. And I got the error message.
I build mlir-rocm-runner following the step on jenkins
`cmake -G Ninja -DCMAKE_BUILD_TYPE=RelWithDebInfo -DMLIR_MIOPEN_DRIVER_ENABLED=1`
`ninja miopen-gen mlir-miopen-driver mlir-rocm-runner ci-performance-scripts`
FWIW, it passes all mlir and mlir-miopen tests.
Can you see what is going wrong here?

Reply:
Hold on, the variable substitution might've led to there being one long argument instead of a bunch of arguments. Perhaps structure `miopen_gen_CMD` as `miopen_gen_CMD=(-a 1 -b 2)` as opposed to `miopen_gen_CMD="-a 1 -b 2"`

Reply:
@jerryyin @krzysz00 Thank you for the suggestions. Actually the error is caused with an extra `', '` in the argument for mlir-rocm-runner.

Reply:
@zhanglx13 LGTM!